server/config/subject_page.proto

```diff
@@ -279,6 +279,14 @@ message Tile {
   // [optional] If set, used instead of main place for the page.
   string place_dcid_override = 11;
 
+  // [optional] used only in ranking tiles
+  // set to true to hide the footer with download / explore more link
+  bool hide_footer = 17;
+
+  // [optional] used only in ranking tiles
+  // whether to show loading spinner during data fetch
+  bool show_loading_spinner = 18;
```

Comment:
I don't actually think show_loading_spinner is something we should add as a config/param ..all tiles should just work the same way in terms of loading treatment and I don't think there's been a case so far where we've wanted a specific loading treatment for a tile .. wdyt?

Reply:
This sounds good to me! Removed as a config/param.